alternative/openrct2.music.mystic/object.json

```
@@ -0,0 +1,22 @@
+{
+    "id": "openrct2.music.mystic",
+    "sourceGame": "official",
+    "authors": ["Skyshard"],
+    "version": "1.0",
+    "objectType": "music",
+    "properties": {
+        "tracks": [
+            {
+                "source": "music/0.flac",
+                "name": "Mystic Winds",
+                "composer": "Skyshard"
+            }
+        ]
+    },
+    "strings": {
+        "name": {
+            "en-GB": "Mystic style",
+			"nl-NL": "Mystiekstijl"
```

```suggestion
            "en-GB": "Mystic style",
			"nl-NL": "Mystiekstijl"
```
Mixed tabs and whitespaces

---

oh oops, notepad++ acting up again, will fix.